arm-network/2016-06-01/swagger/network.json

```diff
@@ -7234,6 +7234,49 @@
       ],
       "description": "IPConfiguration"
     },
+    "ResourceNavigationLinkFormat": {
+      "properties": {
+        "linkedResourceType": {
+          "type": "string",
+          "description": "Gets or sets the resource type of the linked resource"
+        },
+        "link": {
+          "type": "string",
+          "description": "Gets or sets the link to the external resource"
+        },
+        "parentResourceGuid": {
+          "type": "string",
+          "description": "Gets or sets the link to the external resource"
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "Gets or sets Provisioning state of the ResourceNavigationLink resource Updating/Deleting/Failed"
```

Comment:
Was following how other resources are defining provisioningState; All of them don't have readOnly. Deepak?